dvc/logger.py

```diff
@@ -59,12 +59,28 @@ class ColorFormatter(logging.Formatter):
     def format(self, record):
         msg = record.msg.format(*record.args) if record.args else record.msg
         exception, stack_trace = self._parse_exc(record)
-        return ("{prefix}{description}{stack_trace}").format(
+        return ("{asctime}{prefix}{description}{stack_trace}").format(
+            asctime=self.formatTime(record, self.datefmt),
             prefix=self._prefix(record),
             description=self._description(msg, exception),
             stack_trace=stack_trace,
         )
 
+    def formatTime(self, record, datefmt=None):
+        # only show if current level is set to DEBUG
+        # also, skip INFO as it is used for UI
+        if (
+            self._current_level() != logging.DEBUG
+            or record.levelno == logging.INFO
+        ):
+            return ""
+
+        return "{color}{date}{nc} ".format(
+            color=colorama.Fore.GREEN,
+            date=super().formatTime(record, datefmt),
```

Comment:
The default output looks like `2020-02-14 17:13:44,627`. I did not really tried fighting with the format and went with default one. Plus, this supports setting `datefmt` from dictConfig.

Reply:
If we need a different format later on, we can just change in the `dictConfig` anyway. :slightly_smiling_face:

Reply:
Yeah, a bit long for my taste. We could shorten stuff like year (cut it off or shorten to "20"), but it won't make a big deal. I'm ok with leaving it as is.